README.md

```diff
@@ -59,52 +59,74 @@ options:
 > [!NOTE]
 > The scripts on this system relies on the [pddl](https://github.com/AI-Planning/pddl) parser, which can be easily installed via [PyPi](https://pypi.org/project/pddl/) repository (`pip install pddl`). The pddl system relies itself on the [lark](https://lark-parser.readthedocs.io/en/stable/) parsing library. The fond-utils system, however, extends `pddl` to accept single files containing _both_ the domain and the problem instance, and will be extended further to accept labelled outcomes in the effects.
 
-## Example runs
+## Quickstart
```

Comment:
Want the `pip install` path up here in the quick start? Something like this is what I might expect early on...

```shell
pip install fond-utils
fond-utils determinize --input domain.pddl --output det-domain.pddl
```

Reply:
You are suggesting adding those 2 lines at the start of `Quickstart` section? Could be, but would it be too redundant? the pip install path is already above in `Install` section...

Maybe I am not understanding but happy you do what it feels better when you read it, sure!

Reply:
Oh! Nvm. I thought this was up above the install instructions. All good.